src/items/items.hpp

```diff
@@ -435,6 +442,35 @@ class Items {
 		return dummys;
 	}
 
+	static const std::string getAugmentNameByType(AugmentTypes_t augmentType) {
+		phmap::flat_hash_map<AugmentTypes_t, std::string> map = {
```

Comment:
Add it as static to prevent the map from being recreated with each function call

```suggestion
		static phmap::flat_hash_map<AugmentTypes_t, std::string> map = {
```